pkg/init/init.go

```diff
@@ -0,0 +1,76 @@
+package init
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+const PodAnnotationsFileName = "/etc/podinfo/pod_annotations"
+const PodAnnotationsKeyName = "pod-network-controller.istio.io/status"
```

Comment:
we may want to create a separate go file with all the constants, this type of refactoring would limit duplication of codes, see also: handler.go L22 for all the needed constans

Reply:
Its a good idea. Let's add this as an enhancement afterwards